pymeasure/instruments/nf/li5660.py

```diff
@@ -0,0 +1,1665 @@
+
+    def primaryXY_offset_auto(self):
+        """ Automatically sets offsets so that primary PSD's X and Y output become zero at that point, and enables offset adjustment.
+            The function is the same as that of the ":CALC2:OFFS:AUTO:ONCE". Both of these commands work on both X and Y. Automatic setting
+            of just X and Y is not possible."""
+        log.info("Auto offset the primary PSD's X and Y.")
+        self.write(":CALC1:OFFS:AUTO:ONCE")
+
+    def secondaryXY_offset_auto(self):
+        """ Automatically sets offsets so that primary PSD's X and Y output become zero at that point, and enables offset adjustment.
+            The function is the same as that of the ":CALC4:OFFS:AUTO:ONCE". Both of these commands work on both X and Y. Automatic setting
+            of just X and Y is not possible."""
+        log.info("Auto offset secondary PSD's X and Y.")
+        self.write(":CALC3:OFFS:AUTO:ONCE")
+
+    def primaryX_offset_state(self, state="Off"):
+        """ A property that allow user to enable/disable offset for the primary PSD's X output. """
+        log.info("The offset adjustment is set to {}".format(self.name))
+
+        self.calculate1_offset_state = state
+
+    def primaryY_offset_state(self, state="Off"):
+        """ A property that allow user to enable/disable offset for the primary PSD's X output. """
+        log.info("The offset adjustment is set to {}".format(self.name))
+
+        self.calculate2_offset_state = state
+
+    def secondaryX_offset_state(self, state="Off"):
+        """ A property that allow user to enable/disable offset for the primary PSD's X output. """
+        log.info("The offset adjustment is set to {}".format(self.name))
+
+        self.calculate3_offset_state = state
+
+    def secondaryY_offset_state(self, state="Off"):
+        """ A property that allow user to enable/disable offset for the primary PSD's X output. """
+        log.info("The offset adjustment is set to {}".format(self.name))
+
+        self.calculate4_offset_state = state
+
+    def set_calculation_method(self, method="Off"):
+        """ A property that allow user to set the calculation method for measurement value to be displayed and output. """
+        log.info("The calculation method for measurement value is set to {}".format(self.name))
+
+        self.calculation_method = method
+
+    def buffers(self, buffer):
+        buffer_list = {"Buffer1": "BUF1", "Buffer2": "BUF2", "Buffer3": "BUF3"}
+        return buffer_list[buffer]
+
+    def get_buffer(self, buffer="Buffer1", lenght=1, start=0):
+        """ A aquire the floating point data through binary transfer. """
+        buffers = {"Buffer1": "BUF1",
+                  "Buffer2": "BUF2",
+                  "Buffer3": "BUF3"}
+        return self.values(":DATA:DATA? {}, {}, {}".format(buffers[buffer], lenght, start))
```

Comment:
See my comment above; if you have a `self.buffer_list` you don't have to define `buffers` in this method.